docs/01-Introduction.md

```diff
@@ -1,6 +1,89 @@
 # 1.0 - Introduction
 
-## 1.1 What is LORIS?
+### 1.1 What is LORIS-MRI? 
+LORIS-MRI is the backbone of the imaging component that makes up LORIS.
+ It is maintained in a separate repository so that it can be installed
+ on the file server and separated from the web server.
+These documents assume you have some knowledge regarding LORIS and
+a functioning installation. For information regarding LORIS itself,
+please consult the [LORIS wiki][1].
 
-## 1.2 How does LORIS work?
+LORIS-MRI is a set of libraries responsible for the insertion,
+organization, and archival of uploaded imaging datasets.
+ It expects an uploaded, compressed file containing
+a [DICOM][2] study composed of several DICOM files. These DICOM files
+ will be archived on the server and converted to [MINC][3] and [NIfTI][4]
+files. Knowledge of these file formats can be helpful, but are not 
+necessary for using or installing LORIS-MRI.
 
+LORIS-MRI allows you to easily organize and archive your imaging datasets
+and links them with
+ corresponding behavioral data in LORIS. Scans can be viewed and quality
+ controlled through the user's web browser, facilitating collaboration
+ between radiologists, clinicians and researchers.
+
+### 1.2 How does LORIS-MRI work?  ![user_story](images/user_story.png)
+LORIS-MRI allows multiple ways to upload scans, more details about these
+options can be found [here.](05-PipelineLaunchOptions.md) Typically, users
```

Comment:
I would remove the sentence about 'more details about these options can be found here' and just move it to 2 paragraphs later (I will add a comment where I think it makes more sense to put it).
I would just say here:
"LORIS-MRI allows multiple ways to upload scans. But typically, users uploads etc,..."